src/SelfUpdateCommand.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Robo;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem as sfFilesystem;
+
+/**
+ * Update the robo.phar from the latest github release
+ *
+ * @author Alexander Menk <alex.menk@gmail.com>
+ */
+class SelfUpdateCommand extends Command
+{
+    private $command;
+
+    protected $gitHubRepository;
+
+    protected $currentVersion;
+
+    protected $applicationName;
+
+    public function __construct($name = null, $applicationName = null, $currentVersion = null, $gitHubRepository = null)
```

Comment:
`$name` and `$applicaionName` are the same thing, aren't they? Did I miss the place where you changed the [instantiation of the Application instance](https://github.com/consolidation/Robo/blob/1.1.2/src/Robo.php#L270), or did you miss adding a file in your last commit?

Reply:
OIC `$name` is `self:update`.